boot/zephyr/boot_serial_extensions.c

```diff
@@ -21,6 +20,9 @@
 
 BOOT_LOG_MODULE_DECLARE(mcuboot);
 
+#define ZEPHYR_MGMT_GRP_BASIC (MGMT_GROUP_ID_PERUSER - 1)
+#define ZEPHYR_MGMT_GRP_BASIC_CMD_ERASE_STORAGE 0
```

Comment:
This should be including <zephyr/mgmt/mcumgr/grp/zephyr/zephyr_basic.h> not redefining these

Reply:
This is duplicating data and making it redundant/prone to errors if one is changed. @gmarull thoughts on this?

For the command, zephyr_basic.h can at least be included as it has no include requirements

Reply:
MCUboot is independent from Zephyr and may redefine identifiers as it wishes, and actually does so.

Reply:
`boot/zephyr/boot_serial_extensions.c` this is a zephyr file. Neither MCUmgr groups nor commands should not re-registered, or like any duplicate code you have the possibility of a mismatch in the future - this is not a MCUmgr-specific issue